Homework/homework2/task3.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, List
+
+
+def combinations(*args: List[Any]) -> List[List]:
+    """
+    Write a function that takes K lists as arguments.
+
+    Returns all possible lists of K items where the first element is from the first list,
+    the second is from the second and so one.
```

Comment:
Be sure that Grazie plugin is installed into the Pycharm

![image](https://user-images.githubusercontent.com/171597/98706716-3c5e2e00-2390-11eb-8c60-75406dd464c5.png)
![image](https://user-images.githubusercontent.com/171597/98706755-4849f000-2390-11eb-9999-6e5a9fef3844.png)

Reply:
I hadn't this plugin installed, thank you